lib/kb_Metrics/metricsdb_controller.py

```diff
@@ -246,6 +245,7 @@ def _get_activities_from_wsobjs(self, params, token):
                     break
         return {'metrics_result': wsobjs_act}
 
+    @cache_it_json(limit=1024, expire=60 * 60 * 24)
```

Comment:
Doesn't this mean that it would cache ANYTHING for 24 hours?  So if the job finished, this wouldn't expire for 24 hours.  I don't think this is what we want.

Reply:
1. The redis-simple-cache handles the interfacing with redis, including starting and connecting to the redis database.
2. This will cache the result of this function until the specified expiration time, which we can set according to needs or depending on how frequent this service is called.  I set 24 hours here as an example.
Please do elaborate as to what is expected.  I am open to all ideas. Thank you!